### ISSUE TITLE
Incorrect reflect-config.json for nested inner classes

### ISSUE THREAD

**Issue by rsenden** [formkiq/graalvm-annotations-processor#11]
This is very closely related to #9, which was fixed some time ago. Unfortunately, this fix doesn't seem to cover nested inner classes, i.e., in constructs like the following:

```java
@Reflectable
public class Outer {
  @Reflectable
  public static class Inner {
    @Reflectable
    public static class NestedInner {
    }
  }
}
```

The class names listed in reflect-config.json for the above code snippet are as follows:
```
"name": "Outer"
"name": "Outer$Inner"
"name": "Outer.Inner$NestedInner"
```

First two are correct, but I guess the last one should be listed as:
```
"name": "Outer$Inner$NestedInner"
```

Any chance you can have a look at fixing this?

**Comment by mfriesen** [comment]
@rsenden Fixed in v1.4.2

**Comment by rsenden** [comment]
@mfriesen Great, thanks!
